peachjam_search/documents.py

```diff
@@ -31,12 +37,12 @@ class SearchableDocument(Document):
     judges = fields.TextField()
 
     # GenericDocument, LegalInstrument
-    authoring_body = fields.KeywordField()
+    author = fields.KeywordField()
     nature = fields.KeywordField()
 
     class Index:
         # TODO: make this configurable per website
-        name = "africanlii_documents"
+        name = settings.PEACHJAM["APP_NAME"]
```

Comment:
The app name may have spaces, may be uppercased, etc. because it's shown to the user. So I suggest putting a setting such as

```python
PEACHJAM["ES_INDEX"] = os.environ.get("ES_INDEX", slugify(PEACHJAM["APP_NAME"]))
```

that would provide  a reasonable default and allow it to be overridden through an environment variable.

Reply:
And then you don't need to adjust `APP_NAME` is settings.py